docs/undergrad.yaml

```diff
@@ -248,31 +248,30 @@ Affine and Euclidian Geometry:
     equations of affine subspace:
     affine groups:
     affine property:
-    homothetic transformation groups:
-    affinity:
+    group generated by homotheties and translations:
+    transformations fixing a basis of directions: # "affinité", no good English translation
   Convexity:
     convex subsets:
     convex hull of a subset of an affine real space:
     extreme point:
   Euclidean affine spaces:
-    isometries of a Euclidian affine subspace:
-    Euclidian affine space isometry group:
-    Euclidean affine space isometries:
+    isometries of a Euclidean affine space:
+    group of isometries of a Euclidean affine space:
     isometries that do and do not preserve orientation:
-    direct and indirect similarities of the plane:
-    isometric classification in two and three dimensions:
-    angles of vectors: 'inner_product_geometry.angle'
-    angles formed by planes:
+    direct and opposite similarities of the plane:
+    classification of isometries in two and three dimensions:
+    angles between vectors: 'inner_product_geometry.angle'
+    angles between planes:
     inscribed angle theorem:
     cocyclicity:
-    group of isometries stabilizing subset of the plane or space:
+    group of isometries stabilizing a subset of the plane or of space:
     regular polygons:
     metric relations in the triangle:
     using complex numbers in plane geometry:
   Application of quadratic forms to study proper conic sections of the affine euclidean plane:
     focus:
     eccentricity:
-    quadratics on 3 dimensional euclidean affine spaces:
+    quadratics on 3-dimensional Euclidean affine spaces:
```

Comment:
I think this should be quadrics instead of quadratics, right?

Reply:
Yup, just changed to "quadric surfaces" which I think is even clearer.